src/setupfunc.py

```diff
@@ -17,6 +17,8 @@ def do_vcs_install(): pass # --STRIP DURING BUILD
  versionfile_build = myproject/_version.py
  tag_prefix =
  parentdir_prefix = myproject-
+ override_variable = VERSIONEER_OVERRIDE
```

Comment:
Is this the default? I'd like to avoid a situation when I need to patch the project to set this before I can use it.

Reply:
Since versioneer is vendored, there's no real avoiding the patch. But once this exists you could `versioneer install` a newer version and do it without committing.

I'm also okay with making `VERSIONEER_OVERRIDE` an unconfigurable variable name. There's probably not a lot of value to having it vary from project to project.

Reply:
Made `VERSIONEER_OVERRIDE` an unconfigurable variable name.